### PR TITLE
Bug 1869293: Add timestamp to daemon logs

### PR DESCRIPTION
**Pull request by mrogers950**
Also adding a commit to capture daemon logs in the E2E, same code as compliance-operator.

### CODE REVIEW

File: cmd/manager/logcollector.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ const (
 var debugLog bool
 
 func LOG(format string, a ...interface{}) {
-	fmt.Printf(format+"\n", a...)
+	fmt.Printf(fmt.Sprintf("%s: %s\n", time.Now().Format(time.RFC3339), format), a...)
```
Comment (jhrozek) [inline cmd/manager/logcollector.go:54]: Why only add the timestamps to LOG and not DBG?

Comment (mrogers950) [reply cmd/manager/logcollector.go:54]: DBG uses LOG (see below)

**Pull request merged**